src/SUMMARY.md

```diff
@@ -0,0 +1,16 @@
+# Summary
+
+[LazyLedger Specifications](./README.md)
+
+- [Specification](./specs/README.md)
+  - [Architecture](./specs/architecture.md)
+  - [Data Structures](./specs/data_structures.md)
```

Comment:
Didn't we remove the feeHeader btw?

![image](https://user-images.githubusercontent.com/7441766/120081741-6292b380-c0bf-11eb-884a-5d4934f0b304.png)

Reply:
Ah, I just never updated that figure because it's hard to maintain two copies of the same thing 😬. Will fix in a separate PR.

Reply:
I think it's overkill to use tikz for these kind of diagrams btw. Way too time-consuming. Can't these be generated from markdown directly somehow?

Reply:
This figure is actually generated from graphviz/dot, not tikz. Markdown doesn't have any figure support (bare Markdown has almost nothing).

Reply:
How long does it take to create such a diagram in graphviz/dot?

Reply:
> Will fix in a separate PR.

Also, you are making sure that this and the other comments are tracked somewhere (in issues ideally).

Reply:
It takes about 1 minute to update it each time I need to. Just have to add a field or remove one here: https://github.com/lazyledger/lazyledger-specs/blob/aa2abef998f264fb4ba52e2cb026eae712923701/specs/figures/block_data_structures.dot

Reply:
Yeah OK. that indeed looks simple enough. Sorry, it's been a while since I've looked at this file.